flow-typed/interfaces/media-source-adapter.js

```diff
@@ -0,0 +1,9 @@
+//@flow
+declare interface IMediaSourceAdapter {
+  load(): void;
+  destroy(): void;
+  static name: string;
+  static isSupported(): boolean;
+  static canPlayType(mimeType: string): boolean;
+  static createAdapter(engine: IEngine, source: Object, config: Object): IMediaSourceAdapter;
```

Comment:
@yairans please discuss with Oren regards the createAdapter method and dismiss/keep this request accordingly.